src/ucp/api/ucp.h

```diff
@@ -931,13 +931,13 @@ typedef struct ucp_listener_attr {
      * Fields not specified in this mask will be ignored.
      * Provides ABI compatibility with respect to adding new fields.
      */
-    uint64_t              field_mask;
+    uint64_t                field_mask;
 
     /**
-     * The port on which the listener is listening for incoming connection
-     * requests. The port is returned in host byte order.
+     * Sockaddr on which this listener is listening for incoming connection
+     * requests.
      */
-    int                   port;
```

Comment:
This change does break backward compatibility

Reply:
this API was not released yet

Reply:
So you are saying that we didn't do good job the first time around. Once we have why/what/how we can discuss this.